### PR TITLE
`azurerm_healthcare_dicom_service` - extend timeout for healthcare DICOM service

### PR DESCRIPTION
**Pull request by xiaxyi**

### CODE REVIEW

File: internal/services/healthcare/healthcare_dicom_resource.go [hunk 1]
```diff
@@ -30,9 +30,9 @@ func resourceHealthcareApisDicomService() *pluginsdk.Resource {
 		Delete: resourceHealthcareApisDicomServiceDelete,
 
 		Timeouts: &pluginsdk.ResourceTimeout{
-			Create: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Create: pluginsdk.DefaultTimeout(00 * time.Minute),
```
Comment (tombuildsstuff) [inline internal/services/healthcare/healthcare_dicom_resource.go:33]: `0` minutes is going to be problematic, presumably this is meant to be `90`?

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
hi @xiaxyi 

Can you provide a little context on why this needs to be bumped to 3 times the provisioning time, is there an issue this is fixing? In addition, can you update the documentation to account for this?

Thanks!

File: internal/services/healthcare/healthcare_dicom_resource.go (hunk 1 above)
Comment (xiaxyi) [reply internal/services/healthcare/healthcare_dicom_resource.go:33]: Thanks Tom, will do

**Comment by xiaxyi** [comment]
Thanks Tom for the review!

There isn't any api issue relating to the creation of the dicom service, just acctest was failed due to timeout. I made the timeout for DICOM service as the same as FHIR and MedTech service. all of them are created under healthcare workspace.

**Review by tombuildsstuff — APPROVED** [review]
LGTM 👍

**Pull request merged**